assets/js/components/notifications/AdSenseAlerts.js

```diff
@@ -19,18 +19,18 @@
 /**
  * WordPress dependencies
  */
-import { Fragment } from '@wordpress/element';
-import { __, _x } from '@wordpress/i18n';
+import { useEffect, useState } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
-import { useSelect } from 'googlesitekit-data';
+import { useDispatch, useSelect } from 'googlesitekit-data';
 import { CORE_MODULES } from '../../googlesitekit/modules/datastore/constants';
 import { MODULES_ADSENSE } from '../../modules/adsense/datastore/constants';
 import { MODULE_SLUG_ADSENSE } from '@/js/modules/adsense/constants';
-import NotificationAlertSVG from '../../../svg/graphics/notification-alert.svg';
-import BannerNotification from '../notifications/BannerNotification';
+import { NOTIFICATION_AREAS } from '../../googlesitekit/notifications/constants';
+import NotificationFromServer from '../NotificationFromServer';
+import { CORE_NOTIFICATIONS } from '../../googlesitekit/notifications/datastore/constants';
 
 function AdSenseAlerts() {
```

Comment:
Given this component is now simply a wrapper for an effect – which is fine – it would make more sense to migrate to implement via a `DashboardMainEffectComponent` for the module instead. This also ensures the module is active before rendering, but uses the more appropriate dedicated infra for this kind of module-specific effect.

Reply:
Works for me, I'll make that change 👍🏻